### ISSUE TITLE
Feature Request: Add admin panel option to show Rules option to non-admin users

### ISSUE THREAD

**Issue by justinr1234** [wekan/wekan#2217]
Allow admin's an option to show the Rules (IFTTT style behaviours) to non-admin users. Currently this is only available for admin users to see.

It should NOT be enabled for NoComment and CommentOnly roles

**Comment by TiibCD** [comment]
I agree but this option should be available per user ! In the end we need new roles like : superadmin, admin and user
